solidity/DieselPrice.sol

```diff
@@ -4,31 +4,30 @@
     This contract keeps in storage a reference
     to the Diesel Price in USD
 */
-pragma solidity ^0.4.0;
-
-import "github.com/oraclize/ethereum-api/oraclizeAPI.sol";
-
+pragma solidity ^0.4.25;
+import "https://raw.githubusercontent.com/oraclize/ethereum-api/master/oraclizeAPI_0.4.25.sol";
```

Comment:
We'll have to keep the symlink url for the final examples but understand why they are this way in the this PR! 

```suggestion
import "github.com/oraclize/ethereum-api/oraclizeAPI.sol";
```

Reply:
Sure, that makes sense! And yup, that's written in, since it was working code :)